src/inverseRobot.js

```diff
@@ -24,7 +24,25 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+
+  if (robot === {}) {
+    return robot;
+  }
+
+  const ret = {};
+
+  Object.keys(robot).forEach(key => {
+    ret[robot[key]] = key;
+  });
+
+  const retValues = Object.values(ret);
+
+  if (robotValues.length !== retValues.length) {
+    return null;
+  }
```

Comment:
try to use hasOwnProperty method when assigning values to keys in the loop instead of comparing the length at the end